DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tb_utils.py

```diff
@@ -73,6 +76,47 @@ async def tb_clean_config(testbed):
     check_asyncio_results(results, "tb_clean_config")
 
 
+async def tb_post_test_cleanup_qdiscs(testbed):
+    """Removes all non-default qdiscs created during test. Can be used separately or by using `cleanup_qdiscs` fixture."""
+    logger = AppLogger(DEFAULT_LOGGER)
+    dev = await tb_get_all_devices(testbed)
+    dev = dev[0]
+    logger.info("Clearing TC")
+    out = await TcQdisc.show(input_data=[{dev.host_name: [{"options": "-j"}]}], parse_output=True)
+    qdiscs_info = out[0][dev.host_name]["parsed_output"]
+    for qdisc_obj in qdiscs_info:
+        if qdisc_obj["kind"] != "noqueue":
+            await TcQdisc.delete(input_data=[{dev.host_name: [{"dev": qdisc_obj["dev"], "direction": qdisc_obj["kind"]}]}])
+
+
+async def tb_post_test_cleanup_bridges(testbed):
+    """Removes all bridges created during test. Can be used separately or by using `cleanup_bridges` fixture."""
+    logger = AppLogger(DEFAULT_LOGGER)
+    dev = await tb_get_all_devices(testbed)
+    dev = dev[0]
+    logger.info("Clearing bridges")
+    out = await IpLink.show(input_data=[{dev.host_name: [{"link_type": "bridge", "cmd_options": "-j"}]}], parse_output=True)
+    bridges_info = out[0][dev.host_name]["parsed_output"]
+    if bridges_info:
+        await IpLink.delete(input_data=[{dev.host_name: [
+            {"device": bridge_obj["ifname"]} for bridge_obj in bridges_info]}
+        ])
+
+
+async def tb_post_test_cleanup_vrfs(testbed):
```

Comment:
@SerhiyBoikoPLV  Do we need cleanup IPs as well?

Reply:
It would be nice to have, yes. If we only need it for IPv4 tests then we can add it to an ipv4 conftest file. But if other tests will also require it then we should add it to tb_utils.

Reply:
I assume it may be useful for other tests as well. Is there any reference I could use to add IPv4 cleanup function?